### DISCUSSION TITLE
2022-04-28 Gov Committee meeting

### DISCUSSION THREAD

**Discussion by galligan** [cxndao/constellation#4] [Meeting Notes]
## Attendees

- May
- MJ
- Matt Voska

## Notes

- Point: Further governance of the DAO via charter
    - integrity of charter, voting, process
    - what are the goals for the season?
        - ex: 3 proposals passed in season 0
- Lifeblood of DAO - thriving proposal submission
    - A **CIP** - Constellation improvement proposals
        - assigned a number
    - All discussion to be on Discord *before* a vote.
        - Reduce problems beforehand
    - Look at all pre-proposals beforehand
- Snapshot - on-chain voting tool
- Be expert on the charter
    - know who is responsible for what
- Set up base processes for DAO to thrive
- Brian working on town halls - we can give him a hand
- Proposal outline
- “Tokengate a charmverse”
    - way to get notes in a singular place
    - Find a way to organize notes, information, & processes
- Pre-proposals - a channel where ideas can be discussed. When a topic matures to something, make channel for it & create Google Doc around it.
  - Then onto snapshot

## Next steps

- [ ] Review charter
- [ ] thoughts on goals for season 0
- [ ] set next meeting up (Doodle)

**Comment by galligan** [comment]
@mattjoyal I added the meeting notes to a Discussion board so that it can be talked about more if we'd care to!

**Reply by mattjoyal** [reply]
much appreciated.
